docs/quick-start.md

```diff
@@ -100,7 +100,7 @@ argo logs -n argo @latest
 * Open a port-forward so you can access the UI:
 
 ```bash
-kubectl -n argo port-forward deployment/argo-server 2746:2746
+kubectl -n argo port-forward services/argo-server 2746:2746
```

Comment:
I think you can port forward either the deployment or the service, but we want to do the service it is usually singular (e.g., `... port-forward service/argo-server`).

Reply:
Thanks for that. Updated.